templates/puma.rb

```diff
@@ -0,0 +1,20 @@
+# https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server
+
+# The environment variable WEB_CONCURRENCY may be set to a default value based
+# on dyno size. To manually configure this value use heroku config:set
+# WEB_CONCURRENCY.
+workers Integer(ENV.fetch("WEB_CONCURRENCY", 3))
+threads_count = Integer(ENV.fetch("MAX_THREADS", 5))
+threads(threads_count, threads_count)
+
+preload_app!
+
+rackup DefaultRackup
+port ENV.fetch("PORT", 3000)
```

Comment:
@schneems It happens pretty consistently for me. I've pushed an app generate with suspenders and add puma and the config. Setting the port throws the error, link: https://github.com/pedrosmmoreira/puma_double_binding

@croaky, @calebthompson If you feel this should discussed further as an issue in the Puma repo and not here, please let me know.